modules/gltf/src/lib/parse-gltf.js

```diff
@@ -24,16 +24,16 @@ export async function parseGLTF(gltf, arrayBufferOrString, byteOffset = 0, optio
   /** @type {Promise[]} */
   const promises = [];
 
-  if (options.gltf.loadImages) {
-    const promise = loadImages(gltf, options, context);
-    promises.push(promise);
-  }
-
   // Load linked buffers asynchronously and decodes base64 buffers in parallel
   if (options.gltf.loadBuffers && gltf.json.buffers) {
```

Comment:
Nit: I would prefer moving the check for `gltf.json.buffers` into the `loadBuffers` function. 

On this level, the logic should only look at the options, that will make the code simpler to read, not having to take time to think about what it means that the buffers is not defined here.